library/src/main/java/com/google/android/exoplayer2/text/eia608/Eia608Decoder.java

```diff
@@ -159,23 +178,79 @@
     0xC5, 0xE5, 0xD8, 0xF8, 0x250C, 0x2510, 0x2514, 0x2518
   };
 
+  // Maps EIA-608 PAC row numbers to WebVTT cue line settings.
+  // Adapted from: https://dvcs.w3.org/hg/text-tracks/raw-file/default/608toVTT/608toVTT.html#x1-preamble-address-code-pac
+  private static final float[] CUE_LINE_MAP = new float[] {
```

Comment:
Thanks, I didn't know about your implementation.
I'll address all the comments tomorrow.